setup/tank_api_proxy/sgtk/__init__.py

```diff
@@ -19,12 +19,12 @@
 # first look for our parent file
 current_folder = os.path.abspath(os.path.dirname(__file__))
 file_name_lookup = {
-    "linux2": "core_Linux.cfg",
+    "linux": "core_Linux.cfg",
     "win32": "core_Windows.cfg",
     "darwin": "core_Darwin.cfg",
 }
 # Ensure that we get the correct linux key, even if sys.platform returns just "linux"
-platform = "linux2" if sys.platform.startswith("linux") else sys.platform
+platform = "linux" if sys.platform.startswith("linux") else sys.platform
```

Comment:
Hum can we simplify this?
```suggestion
platform = sys.platform
```